e2e/testcafe-devextreme/tests/editors/checkBox/common.ts

```diff
@@ -1,49 +1,79 @@
 import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 import { Selector } from 'testcafe';
+import Guid from 'devextreme/core/guid';
 import url from '../../../helpers/getPageUrl';
 import { createWidget } from '../../../helpers/createWidget';
 import {
   appendElementTo,
-  insertStylesheetRulesToPage, setStyleAttribute,
+  insertStylesheetRulesToPage,
+  setClassAttribute,
+  setStyleAttribute,
 } from '../../../helpers/domUtils';
-import { testScreenshot } from '../../../helpers/themeUtils';
+import { getFullThemeName, testScreenshot } from '../../../helpers/themeUtils';
+
+const valueModes = [false, true, undefined];
 
 const CHECKBOX_CLASS = 'dx-checkbox';
+const READONLY_STATE_CLASS = 'dx-state-readonly';
+const DEFAULT_STATE_CLASS = '';
+const ACTIVE_STATE_CLASS = 'dx-state-active';
+const HOVER_STATE_CLASS = 'dx-state-hover';
+const FOCUSED_STATE_CLASS = 'dx-state-focused';
+const DISABLED_STATE_CLASS = 'dx-state-disabled';
 
-fixture.disablePageReloads`CheckBox`
+fixture.disablePageReloads`CheckBox render`
   .page(url(__dirname, '../../container.html'));
 
-[false, true].forEach((isColumnCountStyle) => {
-  test(`Render ${!isColumnCountStyle ? 'default' : 'with column-count style on container'}`, async (t) => {
-    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+[false, true].forEach((rtlEnabled) => {
+  [20, 30].forEach((iconSize) => {
```

Comment:
Because iconSize is always defined, the size of the checkbox will be the same in both default and compact sizing modes.